my-contacts/src/Body.treat.js

```diff
@@ -0,0 +1,12 @@
+import { style } from 'treat';
+
+export const body = style(theme => ({
+    padding: '2rem',
+    backgroundColor: theme.bodyColor,
+    fontFamily: 'sans-serif',
+    height: '100vh',
```

Comment:
Oooh interesting. I've got the `100vh` there so that in dark mode the entire background becomes black because we can't apply treat themes to the base `body` element. Any thoughts on how I could do this while still filling out the whole page with the black styling?

Reply:
_Sorry for the late reply_

You could look into applying the spacing to children elements or play around with [box-sizing](https://developer.mozilla.org/en-US/docs/Web/CSS/box-sizing) CSS

Reply:
`box-sizing: border-box` has it working! Thanks